tslint.json

```diff
@@ -1,6 +1,7 @@
 {
   "rulesDirectory": [
-    "./tools/tslint-rules/"
+    "./tools/tslint-rules/",
+    "node_modules/codelyzer"
```

Comment:
Will this enforce anything not explicitly listed below?

Reply:
Sorry for the delayed response. No, it won't enforce anything unless you explicit say what you want.